# Ping the robot to verify connection status

---
bool success
string msg
